pkgs/development/libraries/gmime/3.0.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchurl, pkgconfig, glib, gpgme, zlib, libgpgerror, libidn, gobjectIntrospection }:
+
+stdenv.mkDerivation rec {
+  # TODO: libidn support, PGP/MIME support, S/MIME support
+  name = "gmime-3.0.1";
+
+  src = fetchurl {
+    url = "mirror://gnome/sources/gmime/3.0/${name}.tar.xz";
+    sha256 = "001y93b8mq9alzkvli6vfh3pzdcn5c5iy206ml23lzhhhvm5k162";
+  };
+
+  outputs = [ "out" "dev" ];
+
+  buildInputs = [
+    pkgconfig glib gpgme zlib libgpgerror libidn gobjectIntrospection
+  ];
+  nativeBuildInputs = [ pkgconfig ];
+  propagatedBuildInputs = [ glib gpgme zlib libgpgerror libidn ];
+
+  configureFlags = [ "--enable-introspection=yes" ];
+
+  enableParallelBuilding = true;
+
+  meta = {
+    homepage = https://github.com/jstedfast/gmime/;
+    description = "A C/C++ library for creating, editing and parsing MIME messages and structures";
+    maintainers = [ stdenv.lib.maintainers.chaoflow ];
```

Comment:
You can add yourself as maintainer too, if you want.

Reply:
Will do that then.

Reply:
Though I'd like to have some coordination with @chaoflow first.

Reply:
I have not seen anything from chaoflow, since I am maintainer. You can actually only add yourself as maintainer.